internal/indexer/transactions.go

```diff
@@ -40,6 +43,47 @@ func (bc *blockContext) processTransactions(trxHashes *[]string) (err error) {
 			Data: trx.ExtractLogs(),
 		}
 		bc.batch.AddToBatchSingleKey(logs, trx_model.Hash)
+		tp.Go(utils.MakeTask(bc.updateHolderBalances, trx, &err).Run)
+	}
+	return
+}
+
+func (bc *blockContext) updateHolderBalances(trx chain.Transaction) (err error) {
+
```

Comment:
Why do you have empty lines on every method start? :)

Reply:
Old Java things 😆 sorry, will format